modules/module2/module2-04-true_or_false_area_and_lines.qmd

```diff
@@ -0,0 +1,266 @@
+
+```{pyodide}
+fuel_efficiency = alt.Chart(cars).mark_area().encode(
+    x='Year',
+    y='mean(Miles_per_Gallon)').properties(title="Fuel efficiency over time")
+
+fuel_efficiency
+```
+
+:::
+::::
+
+<div id='mcq5'></div> 
+<script>
+    generateQuiz(
+        'mcq5',
+        'Question',
+        'How has the average fuel efficiency changed from the year 1970 to 1982 as a whole?',
+        {
+        'It approximately halved.': 'It looks like efficiency is increasing, not decreasing.',
+        'It approximately doubled.': 'It looks like it went from ~17 miles per gallon to ~31 miles per gallon.',
+        'It approximately tripled.': 'Efficiency is definitely increasing but are you sure it tripled from 1970-1982?',
+        'It approximately quadrupled.': 'This would be a huge accomplishment but are you looking at the correct numbers?'
+        },
+        'It approximately doubled.'
+    );
+</script>
+
+
+Let's now take a look at each country manufacturer. How did the mean fuel efficiency change from the year 1070-1982 for each manufacturing origin?
+
+**Tasks:**<br>
+
+- Plot an appropriate graph that visualizes the change of the mean `Miles_per_Gallon` over time.<br>
+- Assign your plot to an object named `fuel_efficiency_org`.<br>
+- Make sure to give it an appropriate title.
+
+
+```{pyodide}
+#| exercise: fuel_efficiency_graph
+____ = ____
+
+____
+```
+
+```{pyodide}
+#| exercise: fuel_efficiency_graph
+#| check: true
+
+solution = alt.Chart(cars).mark_line().encode( # Or mark_area() would also be acceptable
+    x='Year',
+    y='mean(Miles_per_Gallon)',
+    color = 'Origin').properties(title="Average origin fuel efficiency over time")
+
+assert_chart_equal(solution, result)
+```
+
+:::: { .hint exercise="fuel_efficiency_graph"}
+::: { .callout-note collapse="false"}
+
+## Hint 1
+
+- Are you assigning the plot to an object named`fuel_efficiency_org`?
+- Are you specifying `mean(Miles_per_Gallon)` on the y-axis?
+- Are you specifying `Year` on the x-axis?
+- Are you specifying `colour = 'Origin'`?
+- Are you giving the plot a title using `.properties()`?
+
+:::
+::::
+
+:::: { .solution exercise="fuel_efficiency_graph" }
+::: { .callout-tip collapse="false"}
+
+## Fully worked solution:
+
+```{pyodide}
+fuel_efficiency_org = alt.Chart(cars).mark_line().encode( # Or mark_area() would also be acceptable
+    x='Year',
+    y='mean(Miles_per_Gallon)',
+    color = 'Origin').properties(title="Average origin fuel efficiency over time")
+
+fuel_efficiency_org
+```
+
+:::
+::::
+
+<div id='mcq6'></div> 
+<script>
+    generateQuiz(
+        'mcq6',
+        'Question',
+        'Which origin’s fuel efficiency improved the most between the years 1970 to 1982?',
+        {
+        'Europe': 'Europe has improved, but other origins have improved by greater amounts.',
+        'Japan': 'This country did improve but not the most!',
+        'USA': 'Efficiency is definitely increasing but are you sure it tripled from 1970-1982?'
```

Comment:
```suggestion
        'USA': ''
```

This seems to be the solution from the question above (and was wrong in the old site as well)